### PR TITLE
Improve development docs

### PR DESCRIPTION
**Pull request by AlecRust**
- Use "hst-install-ubuntu.sh" instead of "hst-install.sh" to fix installation issue
- Fix wrong VM name in command
- Increase disk space of dev VM to 15GB
- Add mention of weird mounting issue Multipass sometimes has

### CODE REVIEW

File: docs/docs/contributing/development.md [hunk 1]
```diff
@@ -54,7 +64,7 @@ These instructions use [Multipass](https://multipass.run/) to create the VM. Fee
 
    ```bash
    cd ../install
-   bash hst-install.sh -D /tmp/hestiacp-src/deb/ --interactive no --email admin@example.com --password password123 --hostname demo.hestiacp.com -f
```
Comment (jaapmarcus) [inline docs/docs/contributing/development.md:57]: Without -D /tmp/hestiacp-src/deb/ you are not able install a test build but only "release" branch

Comment (AlecRust) [reply docs/docs/contributing/development.md:57]: With this included, I seem to get errors like this, and the URL doesn't respond in browser:

```sh
[ * ] Configuring Hestia Control Panel...
cp: cannot stat '/usr/local/hestia/install/deb/packages': No such file or directory
sed: can't read /usr/local/hestia/data/packages/*.pkg: No such file or directory
cp: cannot stat '/usr/local/hestia/install/deb/templates/web/unassigned/index.html': No such file or directory
cp: cannot stat '/usr/local/hestia/install/deb/templates/web/skel/document_errors/*': No such file or directory
cp: cannot stat '/usr/local/hestia/install/deb/firewall': No such file or directory
cp: cannot stat '/usr/local/hestia/install/deb/api': No such file or directory
[ * ] Generating default self-signed SSL certificate...
[ * ] Adding SSL certificate to Hestia Control Panel...
[ * ] Enable SFTP jail...
[ * ] Create admin account...
Error: package system doesn't exist
Error: can't create admin user
root@hestia-dev:/home/ubuntu/hestiacp/install#
```

Comment (jaapmarcus) [reply docs/docs/contributing/development.md:57]: Probably the package is missing in /tmp/hestia-src/deb

You need to build the package first then:
./hst_autocompile.sh --hestia --noinstall --keepbuild '~localsrc'

 Or install the release first and then switch to the branch you want with v-update-sys-hestia-git fork branch or 
./hst_autocompile.sh --hestia --noinstall --keepbuild '~localsrc'

After you checked out the correct  branch

**Comment by jaapmarcus** [comment]
We should increase default disk to 15GB 

  -d, --disk <disk>    Disk space to allocate. Positive integers, in bytes, or
                       with K, M, G suffix.
                       Minimum: 512M, default: 5G.

File: docs/docs/contributing/development.md (hunk 1 above)
Comment (jaapmarcus) [reply docs/docs/contributing/development.md:57]: Did you see:
```

[ * ] Installing local package files...
    - hestia core package
    - hestia-php backend package (from apt)
    - hestia-nginx backend package (from apt)

```    

In that case maybe something went wrong?

Comment (jaapmarcus) [reply docs/docs/contributing/development.md:57]: Also if we go install from local package we should use:

 bash hst-install-{os}.sh instead hat-install.sh the last one will just install it from the release branch

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**